### ISSUE TITLE
Build failing when I use this library

### ISSUE THREAD

**Issue by ac-opensource** [SchibstedSpain/Leku#118]
## Description
Gradle error:
`app:transformDexArchiveWithExternalLibsDexMergerForDebug`

## Info Required

- Which version of the library do you actually use?
6.4.5
- Do you have the localization permission granted?
Not relevant
- Are you sending parameters to the activity through the bundle?
Not relevant
- Could you describe what are the actions do you make to raise that error?
Just add the library
- Android monitor shows any log related to the library when the error is shown?
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformDexArchiveWithExternalLibsDexMergerForDebug'.
> com.android.builder.dexing.DexArchiveMergerException: Unable to merge dex
```
- Any other thing that could help me to reproduce the error?
Android Studio 3.0
Gradle 4.3

**Comment by ferranpons** [comment]
@ac-opensource I'm sorry but the latest version of the library, which is the version 3.4.5, is not compatible with the latest google API 26. I need to create a new release version with the update.
I think that I will publish it today or during this week. (it was solved with the PR #117)

Thanks for your submission.

**Comment by ferranpons** [comment]
@ac-opensource we've just published a new version of the library that should solve the issue you have. (v3.6.0)
I close the issue, if you're still experiencing this issue please re-open it again. Many thanks!

Hope it works for you!
